Header header

float64 distance  # m
float64 angle #radians

int32 requested_velocity  # m/s
int32 requested_radius  # m
int32 requested_right_velocity  # m/s
int32 requested_left_velocity  # m/s
int32 encoder_counts_right 
int32 encoder_counts_left
int32 distance_right
int32 distance_left

bool bumper

ImuSensorState imu

int32 motor1
int32 motor2
int32 motor1peak
int32 motor2peak

